main/exercice/exercise.lib.php

```diff
@@ -2250,7 +2199,7 @@ function get_student_stats_by_question($question_id,  $exercise_id, $course_code
  * @param int $session_id
  * @return int
  */
-function get_number_students_question_with_answer_count($question_id, $exercise_id, $course_code, $session_id)
+function get_number_students_question_with_answer_count($question_id, $exercise_id, $course_code, $session_id, $intype = '')
```

Comment:
$intype should be $inType or something like that. What does it mean? (the PHPDoc has not been updated)